tests/integration/test_annotations.py

```diff
@@ -22,21 +22,40 @@
 
 from rest_framework import status
 
-# import fixtures
 from test_utils.emg_fixtures import *  # noqa
 
 
 @pytest.mark.usefixtures('mongodb')
 @pytest.mark.django_db
-class TestAnnotations(object):
+class TestAnnotations:
+
+    @property
+    def _annotation_endpoints(self):
+        return [
+            ['emgapi_v1:analysis-goslim-list', 'go_slim'], 
```

Comment:
Good to see that you add tests for all the new features.

Reply:
We need to add more. :)